From 74c6f4ee052e3030c775ef32b879609c3e232dd7 Mon Sep 17 00:00:00 2001
From: Ignacio R. Morelle <shadowm@wesnoth.org>
Date: Sat, 26 Dec 2009 17:16:23 -0300
Subject: [PATCH 2/2] More progress in scenario 1.05

* Prevent Rishix from getting killed on turn 1 using more ZoC generators
* Added more map decorations
* Fixed the whole village capturing business
* Gave a portrait to the orcish leader
---
 scenarios/05_Bay_of_Tirigaz.cfg |  171 +++++++++++++++++++++++++++-----------
 1 files changed, 121 insertions(+), 50 deletions(-)

diff --git a/scenarios/05_Bay_of_Tirigaz.cfg b/scenarios/05_Bay_of_Tirigaz.cfg
index bd5018c..df0531c 100644
--- a/scenarios/05_Bay_of_Tirigaz.cfg
+++ b/scenarios/05_Bay_of_Tirigaz.cfg
@@ -69,6 +69,7 @@
         canrecruit=yes
         type=Orcish Sovereign
         unrenamable=yes
+        profile="portraits/orcs/warrior.png"
         [modifications]
             {TRAIT_RESILIENT}
             {TRAIT_STRONG}
@@ -239,6 +240,27 @@
         [unit]
             generate_name,random_traits=yes,yes
             type=Saurian Ambusher
+            x,y=22,18
+            facing=ne
+            upkeep=free
+        [/unit]
+        [unit]
+            generate_name,random_traits=yes,yes
+            type=Saurian Soothsayer
+            x,y=27,21
+            facing=n
+            upkeep=free
+        [/unit]
+        [unit]
+            generate_name,random_traits=yes,yes
+            type=Saurian Ambusher
+            x,y=22,22
+            facing=se
+            upkeep=free
+        [/unit]
+        [unit]
+            generate_name,random_traits=yes,yes
+            type=Saurian Ambusher
             x,y=19,18
             facing=se
             upkeep=free
@@ -293,55 +315,108 @@
     [/side]
 
     {STARTING_VILLAGES 2 40}
-    {STARTING_VILLAGES 3 8 }
-    {STARTING_VILLAGES 4 9 }
-    {STARTING_VILLAGES 5 9 }
-    {STARTING_VILLAGES 6 8 }
+    {STARTING_VILLAGES 3 5 }
+    {STARTING_VILLAGES 4 5 }
+
+    #
+    # Some location names
+    #
+    [label]
+        x,y=29,5
+        text= _ "Arthol"
+    [/label]
+    [label]
+        x,y=9,9
+        text= _ "Tirigaz"
+    [/label]
+    [label]
+        x,y=22,24
+        text= _ "Krisval"
+    [/label]
+    [label]
+        x,y=33,13
+        text= _ "Burdyr"
+    [/label]
+
+    {ANIMATED_FOUNTAIN 29 5 }
+    {ANIMATED_FOUNTAIN 9  9 }
+
+    {PLACE_IMAGE ("units/transport/galleon.png~FL(horiz)~RC(magenta>blue)")           6 10}
+    {PLACE_IMAGE ("units/transport/transport-galleon.png~FL(horiz)~RC(magenta>blue)") 8 11}
+    {PLACE_IMAGE ("units/transport/galleon.png~FL(horiz)~RC(magenta>blue)")           11 13}
+    {PLACE_IMAGE ("units/transport/galleon.png~RC(magenta>blue)")                     14 11}
+
+    {PLACE_IMAGE ("units/transport/boat.png")           5  14}
+    {PLACE_IMAGE ("units/transport/boat.png~FL(horiz)") 12 18}
+    {PLACE_IMAGE ("units/transport/boat.png~FL(horiz)") 16 9 }
+
+    {PLACE_IMAGE ("items/straw-bale1.png") 35 14}
+    {PLACE_IMAGE ("items/straw-bale2.png") 32 14}
+    {PLACE_IMAGE ("items/grain-sheaf.png") 37 16}
+    {PLACE_IMAGE ("items/straw-bale2.png") 37 10}
+
+    {PLACE_IMAGE ("items/scarecrow.png")   34 12}
+    {PLACE_IMAGE ("items/scarecrow.png")   33 19}
+
+    {PLACE_IMAGE ("items/archery-target-right.png") 3  7 }
+    {PLACE_IMAGE ("items/archery-target-right.png") 7  5 }
+    {PLACE_IMAGE ("items/archery-target-right.png") 11 6 }
+
+    {PLACE_IMAGE ("scenery/signpost.png") 30 1 }
+
+    [event]
+        name=moveto
+        first_time_only=yes
+        [filter]
+            side,x,y=1,30,1
+        [/filter]
+        [redraw][/redraw]
+        [allow_undo][/allow_undo]
+        [message]
+            speaker=narrator
+            image="scenery/signpost.png"
+            message= _"N: Bisbarl"
+        [/message]
+    [/event]
+
+    {PLACE_IMAGE ("scenery/signpost.png") 24 10}
+
+    [event]
+        name=moveto
+        first_time_only=yes
+        [filter]
+            side,x,y=1,24,10
+        [/filter]
+        [redraw][/redraw]
+        [allow_undo][/allow_undo]
+        [message]
+            speaker=narrator
+            image="scenery/signpost.png"
+            message= _"N: Bisbarl
+NW: Port of Tirigaz
+NE: Arthol"
+        [/message]
+    [/event]
 
-    {ANIMATED_FOUNTAIN 29 5}
+    {PLACE_IMAGE ("scenery/signpost.png") 11 40}
+
+    [event]
+        name=moveto
+        first_time_only=yes
+        [filter]
+            side,x,y=1,11,40
+        [/filter]
+        [redraw][/redraw]
+        [allow_undo][/allow_undo]
+        [message]
+            speaker=narrator
+            image="scenery/signpost.png"
+            message= _"SW: Province of Wyndill"
+        [/message]
+    [/event]
 
     [event]
         name=prestart
-        #
-        # Some location names
-        #
-        [label]
-            x,y=29,5
-            text= _ "Arthol"
-        [/label]
-        [label]
-            x,y=9,9
-            text= _ "Tirigaz"
-        [/label]
-        [label]
-            x,y=22,24
-            text= _ "Krisval"
-        [/label]
-        [label]
-            x,y=33,13
-            text= _ "Burdyr"
-        [/label]
-
-        {PLACE_IMAGE ("units/transport/galleon.png~FL(horiz)~RC(magenta>blue)")           6 10}
-        {PLACE_IMAGE ("units/transport/transport-galleon.png~FL(horiz)~RC(magenta>blue)") 8 11}
-        {PLACE_IMAGE ("units/transport/galleon.png~FL(horiz)~RC(magenta>blue)")           11 13}
-        {PLACE_IMAGE ("units/transport/galleon.png~RC(magenta>blue)")                     14 11}
-
-        {PLACE_IMAGE ("units/transport/boat.png")           5  14}
-        {PLACE_IMAGE ("units/transport/boat.png~FL(horiz)") 12 18}
-        {PLACE_IMAGE ("units/transport/boat.png~FL(horiz)") 16 9 }
-
-        {PLACE_IMAGE ("items/straw-bale1.png") 35 14}
-        {PLACE_IMAGE ("items/straw-bale2.png") 32 14}
-        {PLACE_IMAGE ("items/grain-sheaf.png") 37 16}
-        {PLACE_IMAGE ("items/straw-bale2.png") 37 10}
-
-        {PLACE_IMAGE ("items/scarecrow.png")   34 12}
-        {PLACE_IMAGE ("items/scarecrow.png")   33 19}
-
-        {PLACE_IMAGE ("items/archery-target-right.png") 3  7 }
-        {PLACE_IMAGE ("items/archery-target-right.png") 7  5 }
-        {PLACE_IMAGE ("items/archery-target-right.png") 11 6 }
 
         {OBJECTIVES (
             {OBJECTIVE_VICTORY ( _ "Defeat both enemy leaders")}
@@ -538,11 +613,7 @@
             variable=galas_store
         [/unstore_unit]
 
-        [capture_village]
-            side=1
-            radius=5
-            x,y=$galas_store.x,$galas_store.y
-        [/capture_village]
+        {CAPTURE_VILLAGES 1 $galas_store.x $galas_store.y 5}
 
         {CLEAR_VARIABLE galas_store}
 
-- 
1.6.5

